deployment/docker/alerts.yml

```diff
@@ -187,13 +187,13 @@ groups:
              sum(rate(vm_slow_row_inserts_total[5m])) by(instance)
              /
              sum(rate(vm_rows_inserted_total[5m])) by (instance)
-           ) > 0.5
+           ) > 0.05
```

Comment:
The level has been reduced to 5%, while [the pull request description](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/2210) mentions 10%

Reply:
I did update the commit, but forgot to update PR description. Should be ok now